astropy/table/tests/test_pprint.py

```diff
@@ -161,7 +161,9 @@ def test_format0(self, table_type):
         """
         self._setup(table_type)
         arr = np.arange(4000, dtype=np.float64).reshape(100, 40)
-        lines = table_type(arr).pformat()
+        with conf.set_temp("max_width", None):
+            with conf.set_temp("max_lines", None):
+                lines = table_type(arr).pformat()
```

Comment:
We can chain with comma, right?
```suggestion
        with conf.set_temp("max_width", None), conf.set_temp("max_lines", None):
            lines = table_type(arr).pformat()
```